docs/api/changelog.rst

```diff
@@ -39,22 +43,27 @@ Added
 - Validation for incompatible settings in the :class:`imod.mf6.NodePropertyFlow`
   and :class:`imod.mf6.Dispersion` packages.
 - Checks that only one flow model is present in a simulation when calling
-  :func:`imod.mf6.Modflow6Simulation.regrid_like`, :func:`imod.mf6.Modflow6Simulation.clip_box` or  :func:`imod.mf6.Modflow6Simulation.split`
+  :func:`imod.mf6.Modflow6Simulation.regrid_like`,
+  :func:`imod.mf6.Modflow6Simulation.clip_box` or
+  :func:`imod.mf6.Modflow6Simulation.split`
 - Added support for coupling a GroundwaterFlowModel and Transport Model i.c.w.
-  the 6.4.3 release of MODFLOW. Using an older version of iMOD Python
-  with this version of MODFLOW will result in an error.
+  the 6.4.3 release of MODFLOW. Using an older version of iMOD Python with this
+  version of MODFLOW will result in an error.
 - :meth:`imod.mf6.Modflow6Simulation.split` supports splitting transport models,
   including multi-species simulations.
 - :meth:`imod.mf6.Modflow6Simulation.open_concentration` and
   :meth:`imod.mf6.Modflow6Simulation.open_transport_budget` support opening
   split multi-species simulations.
-  :meth:`imod.mf6.Modflow6Simulation.regrid_like` can now regrid simulations that have 1 or more transport models.
-- added logging to various initialization methods, write methods and dump methods. `See the
-  documentation <https://deltares.github.io/imod-python/api/generated/logging/imod.logging.html>`_
+  :meth:`imod.mf6.Modflow6Simulation.regrid_like` can now regrid simulations
+  that have 1 or more transport models.
+- added logging to various initialization methods, write methods and dump
+  methods. `See the documentation
+  <https://deltares.github.io/imod-python/api/generated/logging/imod.logging.html>`_
   how to activate logging.
 - added :func:`imod.data.hondsrug_simulation` and
   :func:`imod.data.hondsrug_crosssection` data.
-- simulations and models that include a lake package now raise an exception on clipping, partitioning or regridding. 
+- simulations and models that include a lake package now raise an exception on
+  clipping, partitioning or regridding. 
 
 Changed
```

Comment:
some of these changelog messages are only clear to imod-python developers, I think. Maybe we give too much detail?

Reply:
They are indeed quite technical, but I think they are clear enough: We mention the specfic function/class/method in the add/fix/change. When users notice weird behavior in a specific function or class, they can CTRL+F to find a version number that changed something to this function.